docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+# Introduction
+
+This article describes how to update ALZ custom policies and policy initiatives to latest versions of ALZ custom policies. The guidance provided in this document describes manual steps for performing the update, based on a set of specific policies and initiatives, with PowerShell.
+
+> Note: If you are already managing Azure Policies through Infrastructure as Code (IaC), you may not want to be updating your Azure Policies outside of your pipeline. This article assume a manual approach to updating Azure landing zone (ALZ) policies.
+
+> Important: To carry out the instructions below, the operator will require Resource Policy Permissions at the root of the ALZ management group hierarchy.
```

Comment:
is there a link to set this up if this isnt present and also is it worth stating what would happen without this permission?

Reply:
Since this is low-level documentation it's safe to say that we assume that the operator would know how to configure this and is aware of the possible ramifications if it's not present.